frontend/client/src/loading-screen/lettercrap.js

```diff
@@ -86,6 +84,22 @@ function getTextContentWithImageAtSize(
 	const shouldReplaceExisting = () =>
 		!existingText || Math.random() < likelihoodOfChangingExistingText;
 
+	// This code is for scaling the canvas size appropriately for smaller screens
+	let charWidth = 6, charHeight = 10;
+	if (window.innerWidth < 800 && window.innerWidth > 600) {
+		charWidth *= .8;
```

Comment:
My CI makes a good point, in that leading with a decimal could be easy to miss and is not a best practice. Would you mind replacing all the `.8`-type values with `0.8` for clarity?

The anti-pattern identifier is JS-0065 if you would like to read their explanation :) Thanks!